### PR TITLE
Quickstart apps json

### PR DESCRIPTION
**Pull request by andretms**
- Creating apps.json file with sample dependencies

### CODE REVIEW

**Comment by andretms** [comment]
@xerners - can you please review/ merge if this looks ok?

CC: @jmprieur

File: AppCreationScripts/apps.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+  "Sample": {
+    "Title": "Calling Microsoft Graph from an iOS app",
+    "Level": 400,
```
Comment (jmprieur) [inline AppCreationScripts/apps.json]: Is it really a 400? not a 200 ?

**Comment by xerners** [comment]
LGTM.

**Comment by xerners** [comment]
LGTM although Jean-Marc already merged it :-)

On Aug 29, 2018, at 10:51 PM, Jean-Marc Prieur <notifications@github.com<mailto:notifications@github.com>> wrote:


Merged #25<https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FAzure-Samples%2Factive-directory-ios-swift-native-v2%2Fpull%2F25&data=02%7C01%7Cbrandwe%40microsoft.com%7C34e71d41d0c44465318c08d60e3caaae%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636712051051904257&sdata=RN8SVXcOMU42rKfbbcCO2jI3rM0TRjSgbqTXbvgVAcY%3D&reserved=0> into dev.

**Pull request merged**